Makefile

```diff
@@ -8,6 +8,7 @@ ARCHIVE=lxd-$(VERSION).tar
 HASH := \#
 TAG_SQLITE3=$(shell printf "$(HASH)include <dqlite.h>\nvoid main(){dqlite_node_id n = 1;}" | $(CC) ${CGO_CFLAGS} -o /dev/null -xc - >/dev/null 2>&1 && echo "libsqlite3")
 GOPATH ?= $(HOME)/go
+export GO111MODULE=off
```

Comment:
Is the export part actually necessary? I see we set GOPATH without an export for example.

Reply:
Yes, it's necessary. Without it, you'll get
```
go get -t -v -d ./...
go: cannot find main module, but found .git/config in /home/thomas/go/src/github.com/lxc/lxd
	to create a module there, run:
	go mod init
make: *** [Makefile:20: default] Error 1
```